_includes/assets/sass/_variables.scss

```diff
@@ -0,0 +1,48 @@
+/* ==========================================================================
+VARIABLES
+========================================================================== */
+
+// Colors
+$black: #000;
+$white: #ffffff;
+$yellow: #FFDB03;
+$pale-yellow: rgba(252, 247, 215, 0.63);
+$red: #CE132D;
+$grey-background: #E5E5E5;
+$grey-support: #B3BABF;
+
+$body-text: #323E48;
+
+// Custom media queries
+$xs-mobile-down: 340px;
+$sm-mobile: 375px;
+$md-mobile: 480px;
+$mobile: 576px;
+$tablet: 768px;
+$desktop: 992px;
+$lg-desktop: 1200px;
+$xl-desktop: 1440px;
+
+// Fonts
+$primary-font: 'Nunito Sans', sans-serif;
+$secondary-font: 'Source Sans Pro', sans-serif;
+$headings-font: 'Nunito Sans', sans-serif;
+
+// Font weight
+$base-font-weight-bold: 700;
+$base-font-weight-regular: 400;
+$headings-font-weight: 700;
+
+// Font sizes
+$base-font-size: 16px;
+$h1-font-size: 28px;
+$h2-font-size: 26px;
+$h3-font-size: 24px;
+$h4-font-size: 22px;
+
+// Font line heights
+$base-line-height: 16px;
```

Comment:
Sould these line heights scale with the font size? Would it be easier to use em units here?

Reply:
Apologies - the px values are currently placeholders. I intend to do exactly that.

Reply:
.. havent decided to use rem or em yet for these however. If using em for the local sizing makes sense, then yes.